draft-ietf-mls-protocol.md

```diff
@@ -1181,24 +1183,20 @@ struct {
   opaque signature<0..2^16-1>;
 } MLSPlaintextCommitAuthData;
 
-confirmed_transcript_hash_[n] =
-    Hash(interim_transcript_hash_[n-1] ||
+transcript_hash_[n] =
+    Hash(transcript_hash_[n-1] ||
```

Comment:
I don't understand your proposal here.  The transcript hash you define has an odd structure -- it only covers the CommitContent, not the AuthData.  So the confirmation hashes and signatures are not covered.  It's as if the TLS transcript hash didn't include CertificateVerify or Finished.  Is that your intent?

Reply:
In any case, if you're going to remove the interim transcript hash, you need to do more radical surgery.   For example, MLSPlaintextCommitAuthData is no longer needed; likewise much of the prose below.